### DISCUSSION TITLE
SHA256 mismatch when installing RStudio

### DISCUSSION THREAD

**Discussion by Turbonio** [Homebrew/discussions#5002] [Getting started] [answered]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.2.0
ORIGIN: https://github.com/Homebrew/brew
HEAD: ff899e5b3ff6731208564ef502db237e2627e8ba
Last commit: 3 days ago
Core tap JSON: 21 Dec 19:34 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 8
HOMEBREW_SORBET_RUNTIME: set
Homebrew Ruby: 3.1.4 => /opt/homebrew/Library/Homebrew/vendor/portable-ruby/3.1.4/bin/ruby
CPU: octa-core 64-bit arm_firestorm_icestorm
Clang: 15.0.0 build 1500
Git: 2.39.3 => /Library/Developer/CommandLineTools/usr/bin/git
Curl: 8.1.2 => /usr/bin/curl
macOS: 14.0-arm64
CLT: 15.0.0.0.1.1694021235
Xcode: N/A
Rosetta 2: false
```

### Output of `brew doctor`

```shell
Your system is ready to brew.
```

### Description of issue

brew install --cask rstudio        
==> Downloading https://formulae.brew.sh/api/cask.jws.json
#=#=#                                                                          
==> Caveats
rstudio depends on R. The R Project provides official binaries:

  brew install --cask r

Alternatively, the Homebrew-compiled version of R omits the GUI app:

  brew install r

==> Downloading https://download1.rstudio.org/electron/macos/RStudio-2023.12.0-3
Already downloaded: /Users/Om/Library/Caches/Homebrew/downloads/89d4d25168a93d3dd5e5c207078dd7ca2a4a2333b24d90132a0823cfd39c8c85--RStudio-2023.12.0-369.dmg
**Error: SHA256 mismatch
Expected: 55266ead4cd3d3087d2b0a93f20b5e1edf02ffc628350f98409ef8e26c624fb6
  Actual: 4ef1112722c268282bd7a9f78f42607bbc7f11c5b17091edb8e3a7fa737f2eda**
    File: /Users/Om/Library/Caches/Homebrew/downloads/89d4d25168a93d3dd5e5c207078dd7ca2a4a2333b24d90132a0823cfd39c8c85--RStudio-2023.12.0-369.dmg
To retry an incomplete download, remove the file above.

**Comment by miccal** [comment, marked as answer]
Fixed in https://github.com/Homebrew/homebrew-cask/pull/163055
